framework/src/postprocessors/PercentChangePostprocessor.C

```diff
@@ -31,6 +31,9 @@ PercentChangePostprocessor::PercentChangePostprocessor(const InputParameters & p
 void
 PercentChangePostprocessor::initialize()
 {
+  mooseDeprecated("PercentChangePostprocessor is deprecated: instead, ",
+    "please use ChangeOverTimestepPostprocessor using the parameter ",
+    "'compute_relative_change' set to 'true'");
```

Comment:
Move this into a c-tor, `initialize()` can be called as much as once per residual evaluation and this would flood people's terminals.  That would make it annoying enough so they would update their codes, but still... ;-)